### PR TITLE
Add jade sea

### PR DESCRIPTION
**Pull request by EvadLhorg**

### CODE REVIEW

File: packs/JS.json [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+         "faction":"baratheon",
+         "loyal":false,
+         "cost":0,
+         "traits":[
+            "Warship"
+         ],
+         "text":"While you control Dale Seaworth, he gains stealth.\n<b>Action:</b> Kneel and sacrifice Wraith to choose and kneel a location.",
+         "deckLimit":3,
+         "illustrator":"Luciano Neves",
+         "imageUrl": "https://throneteki.ams3.cdn.digitaloceanspaces.com/packs/FH/1_Wraith_ENG.png"
```
Comment (ystros) [inline packs/JS.json:23]: These should all be `JS` instead of `FH`, e.g. https://throneteki.ams3.cdn.digitaloceanspaces.com/packs/JS/1_Wraith_ENG.png

**Review by ystros — CHANGES_REQUESTED** [review]
I looked over the card text and didn't immediately notice anything, just the image URLs need to be JS for Jade Sea instead of FH.

File: packs/JS.json (hunk 1 above)
Comment (EvadLhorg) [reply packs/JS.json]: Thanks, good catch. Fixed.

**Review by stopfstedt — APPROVED** [review]
reviewed on local TDB instance. LGTM.

**Review by ystros — APPROVED** [review]

**Pull request merged**